shared/source/os_interface/linux/drm_neo.cpp

```diff
@@ -659,15 +659,17 @@ std::vector<std::unique_ptr<HwDeviceId>> Drm::discoverDevices(ExecutionEnvironme
 
     do {
         const char *renderDeviceSuffix = "-render";
```

Comment:
```suggestion
        constexpr const char *renderDeviceSuffix = "-render";
```
to allow compile time size calculation

Reply:
fixed in https://github.com/intel/compute-runtime/pull/791/commits/56f6655c6dbe859081d8d932513d9010050d64ab